eng/Version.Details.xml

```diff
@@ -5,4 +5,31 @@
     See https://github.com/dotnet/arcade/issues/2396 for details.
     See https://github.com/dotnet/arcade/blob/master/Documentation/DependencyDescriptionFormat.md#details-file for schema.
   -->
+
+  <!--
+    Following dependencies are required for PVP flow.
```

Comment:
It'd be great to enhance the comment here to say that the versions don't matter for us as we don't have arcade dependency flow.

Reply:
Fixed with https://github.com/NuGet/NuGet.Client/pull/5207/commits/c4e7913463255147da1d0f43577c93c32733e8ad